modules/upgrade/pages/db-migration-13.adoc

```diff
@@ -0,0 +1,123 @@
+[[db-migration-13]]
+= Database Migration from Version 12 to 13
+
+This section covers upgrading the PostgreSQL database from version{nbsp}12 to version{nbsp}13.
+If you are already using PostgreSQL 13, you do not need to perform this migration.
+If you are using version 10, see xref:upgrade:db-migration-12.adoc[].
+If you are using an older version, such as version 9.6, see xref:upgrade:db-migration-10.adoc[].
+
+If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 12 or 13, depending on the underlying operating system. When running on SLES 15 SP3, version 13 should be used. When running on Leap 15.2, version 12 should be used.
+
+
+
+////
+== Background Information
+
+
+* {productname} 3.1.4, SLES 12 SP3, Postgres 9.6
+* {productname} 3.1.11, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 10
+* {productname} 3.2.10, SLES 12 SP4, Postgres 10
+* {productname} 4.0.0, SLES 15 SP1, Postgres 10
+* {productname} 4.1.0, SLES 15 SP2, Postgres 12
+* {productname} 4.2.0, SLES 15 SP3, Postgres 13
+
+////
+
+
+
+
+[[db-migration-13-prepare]]
+== Prepare to Upgrade
+
+Before you begin the upgrade, prepare your existing {productname} Server and create a database backup.
+
+PostgreSQL stores data at [path]``/var/lib/pgsql/data/``.
+
+.Procedure: Preparing to Upgrade
+
+. Check the active PostgreSQL version:
++
+----
+psql --version
+----
++
+If you are using PostgreSQL{nbsp}12, you can upgrade to PostgreSQL{nbsp}13.
++
+If you are already using PostgreSQL version 13, you do not need to perform this migration.
+. Check the active smdba version:
++
+----
+rpm -q smdba
+----
++
+// CHECKIT!
++
+PostgreSQL{nbsp}13 requires ``smdba`` version 1.7.6 or later.
+
+. Perform a database backup.
+    For more information on backing up, see xref:administration:backup-restore.adoc[].
+
+
+
+[[db-migration-13-upgrade]]
+== Upgrade PostgreSQL
+
+[WARNING]
+====
+Always create a database backup before performing a migration.
+====
+
+PostgreSQL upgrades can be performed in two ways: a regular upgrade, or a fast upgrade:
+
+A regular upgrade will create a complete copy of the database, so you will need double the existing database size of space available.
+Regular upgrades can take a considerable amount of time, depending on the size of the database and the speed of the storage system.
+
+A fast upgrade only takes a few minutes, and uses almost no additional disk space.
+However, if a fast upgrade fails, you must restore the database from the backup.
+A fast upgrade reduces the risk of running out of disk space.
```

Comment:
... but increase the risk of data lose when a backup does not exist or cannot be replayed